docs/concepts/packaging-types.adoc

```diff
@@ -55,20 +55,19 @@ $ java -jar myapp-hollow-swarm.jar myapp.war
 
 === Pre-Built Hollow Jars
 
-WildFly Swarm ships three pre-built hollow jar types, for different use cases:
+WildFly Swarm ships the following pre-built hollow JARs:
```

Comment:
Avoid explicitly referring to how many things there are since those tend to change often.

The last part is IMO superfluous because the description of the JARs is clear about the use case.